scram/route_manager/api/views.py

```diff
@@ -76,19 +77,24 @@ def perform_create(self, serializer):
             raise PrefixTooLarge()
 
         # Make sure this client is authorized to add this entry with this actiontype
-        client_uuid = self.request.data["uuid"]
-        authorized_actiontypes = Client.objects.filter(uuid=client_uuid).values_list(
-            "authorized_actiontypes__name", flat=True
-        )
-        authorized_client = Client.objects.filter(uuid=client_uuid).values(
-            "is_authorized"
-        )
-        if not authorized_client or actiontype not in authorized_actiontypes:
-            logging.debug(f"Client {client_uuid} actiontypes: {authorized_actiontypes}")
-            logging.info(
-                f"{client_uuid} is not allowed to add an entry to the {actiontype} list"
+        if self.request.data.get("uuid", False):
```

Comment:
I think the default for get is just None, which would also make this if False.

Reply:
cleaned up this line (and fixed broken permissions override).